tailwind.config.js

```diff
@@ -56,10 +63,15 @@ module.exports = {
         'brand-gray-dark': '#464545',
         'brand-gray-darkest': '#2D2D2D',
         'brand-eggshell': '#f8f8f8',
+        'gray-primary': '#242323',
+        'hubblr-purple': '#5d1c90',
+        'hubblr-turkis': '#4ad0d9',
+        'hubblr-link-gradient-1': '#4299e1',
```

Comment:
Hier fände ich es besser die Farben direkt zu benennen. Unter hubblr-link-gradient-1 kann man sich weniger vorstellen auf einen Blick und die Farbe könnte ja auch noch woanders genutzt werden in der Zukunft.